src/rprblender/export/mesh.py

```diff
@@ -315,12 +315,26 @@ def sync(rpr_context: RPRContext, obj: bpy.types.Object, **kwargs):
         rpr_context.create_empty_object(obj_key)
         return
 
-    rpr_shape = rpr_context.create_mesh(
-        obj_key,
-        data.vertices, data.normals, data.uvs,
-        data.vertex_indices, data.normal_indices, data.uv_indices,
-        data.num_face_vertices
-    )
+    deformation_data = rpr_context.deformation_cache.get(obj_key)
+    if deformation_data and np.any(data.vertices != deformation_data.vertices) and \
+            np.any(data.normals != deformation_data.normals):
+        vertices = np.concatenate((data.vertices, deformation_data.vertices))
+        normals = np.concatenate((data.normals, deformation_data.normals))
+        rpr_shape = rpr_context.create_mesh(
+            obj_key,
+            np.ascontiguousarray(vertices), np.ascontiguousarray(normals), data.uvs,
+            data.vertex_indices, data.normal_indices, data.uv_indices,
+            data.num_face_vertices,
+            {pyrpr.MESH_MOTION_DIMENSION: 2}
+        )
+    else:
+        rpr_shape = rpr_context.create_mesh(
+            obj_key,
+            data.vertices, data.normals, data.uvs,
+            data.vertex_indices, data.normal_indices, data.uv_indices,
+            data.num_face_vertices
+        )
```

Comment:
Should't there be the None passed as a mesh_info value?
The other way would be to add a default "None" value to it in the rpr_context.create_mesh method header.

Reply:
None is not needed here, it is ok here.